Extension/lib/content-script/adguard-content.js

```diff
@@ -17,8 +17,5 @@
 
 /**
  * Global object for content scripts
- * @type {{}}
  */
-var adguardContent = (function () { // jshint ignore:line
```

Comment:
Why can't we create it in the `content-script.js`? As I see this file is already used in every content script.